source/TARGET_K66F/hardware_init.c

```diff
@@ -39,7 +39,7 @@ void initialize_enet_hardware(void)
 
 #ifndef FEATURE_UVISOR
     /* Disable MPU only when uVisor is not around. */
-    MPU->CESR &= ~MPU_CESR_VLD_MASK;
+    SYSMPU->CESR &= ~SYSMPU_CESR_VLD_MASK;
```

Comment:
Looking at this, can t his code be made backward-compatible? If using older version, use MPU, otherwise use SYSMPU? 

I beleive this whould resolve the cyclic dependency, and make this PR to be integrated